.github/workflows/ci-client-cyclic-deps-check.yml

```
@@ -55,60 +55,90 @@ jobs:
+
+            # Only output the "Dependencies removed:" header if there are any removed dependencies.
+            if [[ -n "$removed" ]]; then
+              echo "Dependencies removed:" >> diff.txt
+              echo "$removed" >> diff.txt
+            fi
+
+            cat diff.txt
       # Comment on the PR if cyclic dependencies are found
       - name: Comment the result on PR
         if: steps.compare-deps.outputs.has_more_cyclic_deps == 'true' && steps.changed-files.outputs.any_changed == 'true'
         uses: actions/github-script@v3
```

actionlint (1.7.4)

the runner of "actions/github-script@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)